string joint
---
int16 ticks